client/src/connection/itc/index.ts

```diff
@@ -280,22 +300,80 @@ export class ITCSession extends Session {
    */
   private onShellStdErr = (chunk: Buffer): void => {
     const msg = chunk.toString();
-    console.warn("shellProcess stderr: " + msg);
-    this.clearPassword();
+
+    const errorMessage = this._errorParser.processLine(msg);
+    if (!errorMessage) {
+      return;
+    }
+
     this._runReject(
-      new Error(
-        l10n.t(
-          "There was an error executing the SAS Program. See [console log](command:workbench.action.toggleDevTools) for more details.",
-        ),
-      ),
+      new Error(this.fetchHumanReadableErrorMessage(errorMessage)),
     );
+
     // If we encountered an error in setup, we need to go through everything again
-    if (/Setup error/.test(msg)) {
+    const fatalErrors = [/Setup error/, /powershell\.exe: command not found/];
+    if (fatalErrors.find((regex) => regex.test(errorMessage))) {
+      // If we can't even run the shell script (i.e. powershell.exe not found),
```

Comment:
if the system language is not English, it may get translated "not found", isn't it? not sure if there's any better way to check this

Reply:
That's a good point. We _could_ just isolate this to check `powershell\.exe`. I wonder if that'd be problematic. Fwiw, the _only_ error I've seen with `powershell\.exe` as part of the message is this one 🤷

Reply:
Okay, I simplified this in https://github.com/sassoftware/vscode-sas-extension/pull/842/commits/58babb4f1a20be6cc3590605cf35b0bfa2a5bbe4. It _seems_ to work well for running code successfully and w/ errors. 

That said, if we think it's _too_ generic, I can roll it back and just let this error fall through to the "default" error